src/uproot/extras.py

```diff
@@ -147,27 +146,9 @@ def lzma():
     Imports and returns ``lzma`` (which is part of the Python 3 standard
     library, but not Python 2).
     """
-    try:
-        import lzma
-    except ImportError:
-        try:
-            import backports.lzma as lzma
-        except ImportError:
-            raise ImportError(
-                """install the 'lzma' package with:
-
-    pip install backports.lzma
-
-or
-
-    conda install backports.lzma
+    import lzma
 
-or use Python >= 3.3."""
-            )
-        else:
-            return lzma
-    else:
-        return lzma
+    return lzma
```

Comment:
This function is kind of pointless now, don't know if you want to deprecate it.

Reply:
I had that in the back of my mind: we don't have to get `lzma` from extras.py anymore; it can be a top-of-the-module import. I think it only appears in compression.py.